NuRadioMC/SignalProp/analyticraytracing.py

```diff
@@ -417,8 +417,12 @@ def __init__(self, medium, attenuation_model=None,
             else:
                 self.__logger.info('Using C++ raytracer')
         else:
-            self.__logger.warning('C++ raytracer is not available. Using Python raytracer.')
-            self.__logger.warning("check NuRadioMC/NuRadioMC/SignalProp/CPPAnalyticRayTracing for manual compilation")
+            if use_cpp:
+                self.__logger.error('C++ raytracer was explicitly requested, but is not available. Abort.... Either fix the compilation or set use_cpp to False')
+                raise RuntimeError('C++ raytracer was explicitly requested, but is not available. Abort.... Either fix the compilation or set use_cpp to False')
```

Comment:
[nitpick] is it worth still pointing the user towards `NuRadioMC/NuRadioMC/SignalProp/CPPAnalyticRayTracing` in this case?